dashboard/plugins/s3_usage/s3_stats.py

```diff
@@ -0,0 +1,193 @@
+import boto3
+import sqlite3
+import fcntl
+import os
+import time
+import threading
+import re
+from collections import defaultdict
+from concurrent.futures import ThreadPoolExecutor
+from dataclasses import dataclass
+from dashboard.utils import sizeof_fmt
+
+class S3Stats:
+    ORDER_DEFINITION = {
```

Comment:
This enum is only used in this file. I think we can extract it from class and move it right under imports?

Reply:
Actually, it's even used only in one method, but I put it here as a "class constant static field".